test/functional/tests/fault_injection/test_standby_activate_different_cache_path.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cli_messages import cache_activated_successfully, check_stdout_msg
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.filesystem.symlink import Symlink
+from test_utils.size import Size, Unit
+from api.cas.cache_config import CacheLineSize
+from api.cas.cache import CacheStatus
+from test_tools.dd import Dd
+from test_tools.fs_utils import check_if_symlink_exists
+from test_utils.os_utils import sync
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+def test_standby_activate_different_cache_path(cache_line_size):
+    """
+    title: test_standby_activate_different_cache_path
+    description: |
+        Symlink-activated CAS partitions distinction test.
+    pass_criteria:
+      - Cache is successfully activated with symlink on second partition.
+      - Cache is present and running.
+    """
+    with TestRun.step("Prepare two partitions with the same size."):
+        cache_dev = TestRun.disks["cache"]
+        cache_dev.create_partitions([Size(500, Unit.MebiByte)] * 2)
+        active_cache_dev = cache_dev.partitions[0]
+        standby_cache_dev = cache_dev.partitions[1]
+        cache_id = 1
+
+    with TestRun.step("Start a regular cache instance on one of the partitions and stop it."):
+        active_cache_dev = casadm.start_cache(
+            active_cache_dev, cache_line_size=cache_line_size, force=True
+        )
+        active_cache_dev.stop()
+
+    with TestRun.step("Create a symbolic link to the second partition."):
+        link_path = "/dev/disk/by-id/standby_cache_dev_link"
+        symlink = Symlink.create_symlink(link_path, standby_cache_dev.path, True)
+        if check_if_symlink_exists(link_path):
+            TestRun.LOGGER.info("Symlink created successfully.")
```

Comment:
Isn't the test supposed to fail in case the symlink doesn't exist?

Reply:
This comment is still relevant. You need to call `TestRun.fail()` instead of `TestRun.LOGGER.info()`